docs/settings.rst

```diff
@@ -0,0 +1,70 @@
+#############
+Settings Page
+#############
+
+
+On this page, it is possible to select options that will persist beyond your current browsing session. Your  choices are saved in the browser using a technology called IndexedDB. This allows you, for example, to choose 50 as the number of results to be included on the website data tables, and then this value will be remembered and the next time you visit our website, it will use 50 records for all the data tables.
```

Comment:
Replace "On this page" by "On the settings page" ( and add link to the InterPro website settings page)